test/e2e/httpproxy/request_redirect_test.go

```diff
@@ -43,15 +43,6 @@ func testRequestRedirectRuleNoService(namespace string) {
 	})
 }
 
-func testRequestRedirectRule(namespace string) {
-	Specify("redirects can be specified on route rule", func() {
-		t := f.T()
-
-		proxy := getRedirectHTTPProxy(namespace, false)
-		doRedirectTest(namespace, proxy, t)
-	})
-}
-
```

Comment:
Per previous comment, this needs to be added back along with the call to it in `httpproxy_test.go`.

Reply:
Ah, got it, that makes sense.